.circleci/config.yml

```
@@ -124,10 +124,23 @@ commands:
                 # For whatever reason we have to install pytorch first. If it isn't
                 # it installs the 1.4 cpuonly version. Which is no good.
                 #conda install -q -y pytorch=1.12 torchvision pytorch-cuda=11.3 -c pytorch -c nvidia
-                conda install pytorch==1.12.1 torchvision==0.13.1 cudatoolkit=11.3 -c pytorch
-                conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis pytest-mock
+                conda install -y pytorch=1.13 torchvision cudatoolkit=11.6 pytorch-cuda=11.6 -c pytorch -c nvidia
+                conda install -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis pytest-mock
                 pip install pytest-sugar pytest-xdist pytest-benchmark opencv-python cython mock
               fi
+      - run: &validate_pytorch_installation
+          name: Validate Pytorch Installation
+          no_output_timeout: 5m
+          command: |
+              # Check that pytorch is installed with CUDA.
+              export PATH=$HOME/miniconda/bin:$PATH
+              . activate habitat;
+              $(python -c 'import torch; torch.cuda.set_device(0)')
```

```suggestion
              python -c 'import torch; torch.cuda.set_device(0)'
```
Why isn't that sufficient to propagate the error code?